phpcs-sniffs/Formidable/Sniffs/CodeAnalysis/SimplifyIfReturnSniff.php

```
@@ -0,0 +1,257 @@
+		}
+
+		// Get the condition and return values to check line length.
+		$conditionStart = $phpcsFile->findNext( T_OPEN_PARENTHESIS, $stackPtr + 1 );
+		$conditionEnd   = $tokens[ $conditionStart ]['parenthesis_closer'];
+		$condition      = trim( $phpcsFile->getTokensAsString( $conditionStart + 1, $conditionEnd - $conditionStart - 1 ) );
+
+		$firstReturnValue  = trim( $phpcsFile->getTokensAsString( $firstStatement + 1, $returnEnd - $firstStatement - 1 ) );
+		$secondReturnValue = trim( $phpcsFile->getTokensAsString( $afterIf + 1, $secondReturnEnd - $afterIf - 1 ) );
+
+		// Check if condition starts with ! and flip if so.
+		$isNegated = false;
```

PHPMD (2.15.0)

Avoid unused local variables such as '$isNegated'. (undefined)

(UnusedLocalVariable)